### ISSUE TITLE
Customization of the alert shown when there is no google play services

### ISSUE THREAD

**Issue by egesu** [dpa99c/cordova-plugin-request-location-accuracy#31]
I get an alert like `<appname> won't run without Google Play Services, which are not supported by your device`. This is an annoying message for people who don't like installing google apps in their phone. Also the app runs, just some features won't work. Can I customize it (localize it for example) or completely disable it?

**Comment by dpa99c** [comment]
TL;DR I don't intend the plugin to support Android devices without Google Play Services - if you want to do this, fork the plugin and modify as necessary.

This plugin won't work without the Google Play Services location library installed on the device - it relies on that API in order to enable location services from within the app. There are, in reality, very few real users running Android devices which do not have Google Play Services installed, and if they do then they do so at their own risk, since Google does not support Android devices which do not have this because it's considered a core library.

The message that is displayed is not under the control of this plugin - it comes from [Google API Availability](https://developers.google.com/android/reference/com/google/android/gms/common/GoogleApiAvailability) which the plugin [makes use of](https://github.com/dpa99c/cordova-plugin-request-location-accuracy/blob/master/src/android/RequestLocationAccuracy.java#L514) in order to display the dialog in the manner [recommended by Google](https://developers.google.com/android/guides/setup#ensure_devices_have_the_google_play_services_apk) and is loosely based on [this Google example](https://github.com/googlesamples/google-services/blob/master/android/gcm/app/src/main/java/gcm/play/android/samples/com/gcmquickstart/MainActivity.java#L107).
